posts/2021/10/jupyter-community-calls.md

```diff
@@ -0,0 +1,102 @@
+<!--
+.. title: A year of Jupyter community calls
+.. slug: jupyter-community-calls
+.. date: 2021-11-10
+.. author: Isabela Presedo-Floyd, Tony Fast
+.. tags: Jupyter, Community
+.. category:
+.. link:
+.. description:
+.. type: text
+-->
+
+# Jupyter community calls
+
+A framing for open source is that the software and code are kernels of community.
+The code, and its abstractions, unite developers and their patrons; 
+a struggle for growing/evolving open communities is to make sure these groups remain connected. 
+A lot of us showed up for the code, but hung around for the community. 
+We'll continue this post talking about the monthly Jupyter community calls, 
+and how they help all [jovyans](https://jupyter.readthedocs.io/en/latest/community/content-community.html#what-is-a-jovyan), Project Jupyter's pet name for their developers and users, stay connected.
+
+
+## Why community calls?
+
+  > A lot of us showed up for the code, but hung around for the community. 
+
+[Project Jupyter has a rich community calendar](https://jupyter.org/events) spanning different projects in the ecosystem. 
+Although these events are open to all, the meetings can appear daunting and exclusive. 
+
+This year, Quansight Labs took on the responsibility of reviving an existing series of events: 
+the Jupyter community calls. 
+The original monthly series, starting in [March 2019][first], gave the stage to those using Jupyter daily. 
+These events started to created space for Jupyter's patrons to demonstrate how they use Jupyter as a computational tool.
+
+Jupyter is a rapidly moving piece of technology, and staying up to date with the latest 🔥 takes significant labor. 
```

Comment:
I feel like I miss the meaning of the fire emoji here - if it's possible I feel like including the emoji but also the word it should mean might make it easier to understand.

Reply:
i clarified this in a recent commit. thanks for helping me get rid of my slang.